generator/src/main/java/com/algorand/sdkutils/generators/OpenApiParser.java

```diff
@@ -471,14 +487,20 @@ public void generateQueryMethods() throws IOException {
      */
     @Deprecated
     public OpenApiParser(JsonNode root) {
-        this.javaMode = true;
```

Comment:
How do we select Go or Java generator now?

Reply:
we don't :-) why should we? it should always be the same :-)
if needed, can easily add another parameter to this, and not subscribe java or go listener.